app/src/app/[lng]/[inventory]/InventoryCalculationTab.tsx

```diff
@@ -0,0 +1,165 @@
+"use client";
+
+import { SectorCard } from "@/components/Cards/SectorCard";
+import { SegmentedProgress } from "@/components/SegmentedProgress";
+import { CircleIcon } from "@/components/icons";
+import { useTranslation } from "@/i18n/client";
+import { formatPercent } from "@/util/helpers";
+import {
+  InventoryProgressResponse,
+  InventoryResponse,
+  SectorProgress,
+} from "@/util/types";
+import {
+  Box,
+  Center,
+  Heading,
+  Spinner,
+  Tag,
+  TagLabel,
+  TagLeftIcon,
+  Text,
+} from "@chakra-ui/react";
+import { Trans } from "react-i18next/TransWithoutContext";
+import { TabHeader } from "@/app/[lng]/[inventory]/TabHeader";
+
+function sortSectors(a: SectorProgress, b: SectorProgress): number {
+  const refA = a.sector.referenceNumber;
+  const refB = b.sector.referenceNumber;
+  if (!refA || !refB) {
+    return 0;
+  } else if (refA < refB) {
+    return -1;
+  } else if (refA > refB) {
+    return 1;
+  }
+  return 0;
+}
+
+export default function InventoryCalculationTab({
+  lng,
+  inventory,
+  isUserInfoLoading,
+  isInventoryProgressLoading,
+  inventoryProgress,
+}: {
+  lng: string;
+  inventory?: InventoryResponse;
+  isUserInfoLoading?: boolean;
+  isInventoryProgressLoading?: boolean;
+  inventoryProgress?: InventoryProgressResponse;
+}) {
+  const { t } = useTranslation(lng, "dashboard");
+  let totalProgress = 0,
+    thirdPartyProgress = 0,
+    uploadedProgress = 0;
+  if (inventoryProgress && inventoryProgress.totalProgress.total > 0) {
+    const { uploaded, thirdParty, total } = inventoryProgress.totalProgress;
+    totalProgress = (uploaded + thirdParty) / total;
+    thirdPartyProgress = thirdParty / total;
+    uploadedProgress = uploaded / total;
+  }
+  return (
+    <>
```

Comment:
I think this has only 1 child, so do you need the outermost <> here?

Reply:
We needed because the first thing we have are {}